pubsub/azurepubsub/example_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azurepubsub_test
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"gocloud.dev/pubsub"
+	"gocloud.dev/pubsub/azurepubsub"
+)
+
+var (
+	// See docs below on how to provision an Azure Service Bus Namespace and obtaining the connection string.
+	// https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues
+	connString = os.Getenv("SERVICEBUS_CONNECTION_STRING")
```

Comment:
Same here, moving this var and comment inline into each of the two examples so users see it.